freeipa/src/main/java/com/sequenceiq/freeipa/client/FreeIpaClientBuilder.java

```diff
@@ -231,7 +231,10 @@ private CookieAndStickyId connect(String user, String pass, String apiAddress, i
                         response.getStatusLine().getStatusCode()));
             }
         }
-        Cookie sessionCookie = cookieStore.getCookies().stream().filter(cookie -> "ipa_session".equalsIgnoreCase(cookie.getName())).findFirst().get();
+        Cookie sessionCookie = cookieStore.getCookies().stream()
```

Comment:
Lets add this comment block from the canonical freeipa client for reference

> A cookie string may contain multiple cookies, parse the cookie
 string and return the last cookie in the string matching the
 cookie name or None if not found.
 method which iterates over what parse() returns looking for
 the specific cookie.
 When cookie_name appears more than once the last instance is
 returned rather than the first because the ordering sequence
 makes the last instance the current value.